### ISSUE TITLE
invalid_grant

### ISSUE THREAD

**Issue by coolbox** [m4i/omniauth-runkeeper#3]
Hi,

Thanks for this gem. I'm having an issue with it though and wondered if you could help please?

Once i've logged into RK and the callback phase is initiated i get: `Authentication failure! invalid_credentials: OAuth2::Error, invalid_grant:`

From my logs i can see:
```
23:32:41 web.1  | I, [2015-10-26T23:32:41.303173 #83218]  INFO -- omniauth: (runkeeper) Callback phase initiated.
23:32:42 web.1  | E, [2015-10-26T23:32:42.325118 #83218] ERROR -- omniauth: (runkeeper) Authentication failure! invalid_credentials: OAuth2::Error, invalid_grant: 
23:32:42 web.1  | {"error":"invalid_grant"}
```
Do you know i might be getting this error?

Thanks

**Comment by maxime** [comment]
Hi @coolbox, I suddenly have the same issue as yours. The problem comes from the fact that oauth2 supplies the code parameter in the redirect_uri by default. Overriding the query_string method works for now.

Maxime

**Comment by m4i** [comment]
Thank you.
It was maybe fixed by #4.
